src/Mod/Draft/draftguitools/gui_edit_base_object.py

```diff
@@ -61,17 +61,36 @@ def update_object_from_edit_points(self, obj, node_idx, v, alt_edit_mode=0):
         """
         pass
 
-    def get_edit_point_context_menu(self, obj, node_idx):
-        """ Return a list of Draft_Edit context menu actions.
+    def get_edit_point_context_menu(self, edit_command, obj, node_idx):
+        """ Get the context menu associated to edit points (user is over an editpoint)
+
+        Return a dictionary of menu labels and associated functions:
+            return {
+                "add point": lambda: self.handle_add_point(edit_command, obj, node_idx),
```

Comment:
Maybe it is better to use a list of tuples here, instead of a dictionary after all. Because:
 - A list is ordered and (until CPython 3.6, dictionaries are not)
 - A list is probably more lightweight
 - We only need to iterate over the actions, never look up an action by its label, so there is no need for a dict.
 
 
 This would look like e.g.

```python
return [
    ("add point", lambda: self.handle_add_point(edit_command, obj, node_idx)),
]
```

Reply:
Done!